otterai/main.py

```diff
@@ -354,28 +350,25 @@ def main():
     pr_number = int(os.getenv('GITHUB_EVENT_NUMBER'))
     extra_prompt = os.getenv('INPUT_EXTRA_PROMPT', '')
     workspace = os.getenv('GITHUB_WORKSPACE', '.')
-
+    
     logging.info("🦦 Dr. OtterAI starting code review...")
```

Comment:
📜 Consider logging the `repo` and `pr` details for better traceability during debugging. 🔍